cinch/roles/jenkins_master/tasks/iptables.yml

```diff
@@ -40,12 +40,7 @@
     jump: REDIRECT
     to_ports: "{{ https_enabled | ternary(8443, 8080) }}"
     state: present
-  register: real_address_rule
-
-- name: save iptables
-  shell: service iptables save
-  changed_when: false
-  when: forward_rule.changed or loopback_rule.changed or real_address_rule.changed
+  notify: save iptables
```

Comment:
You are right, my mistake. I'd prefer adding a meta task to flush handlers here and in other roles where necessary, instead of going back to when.changed (which breaks lint rules).